spec/themis_spec.vim

```diff
@@ -0,0 +1,48 @@
+source spec/support/helpers.vim
+runtime! plugin/vim-themis
+
+describe "Themis"
+
+  before
+    let g:test#viml#themis#executable =
+          \ getcwd() . '/.vim-flavor/deps/thinca_vim-themis/bin/themis'
```

Comment:
Since the `g:test#viml#themis#executable()` function will not get executed, we can actually assign this to whatever we want. So let's assign this to just `let g:test#viml#themis#executable = 'themis'`.

Then in tests we can replace assertions

```vim
Expect g:test#last_command == g:test#viml#themis#executable . ' math.vim'
```

with

```vim
Expect g:test#last_command == 'themis math.vim'
```